snapshots/lgbt_rights/2024-06-11/criminalization_mignot.py

```diff
@@ -0,0 +1,25 @@
+"""Script to create a snapshot of dataset."""
```

Comment:
maybe brief explanation on how to get the data?

I suspect it'd be sth like 

```
python table_extract.py

Reply:
Oh yes, I explained it in table_extract. I will write a text here